src/openassetio-python/package/openassetio/test/manager/harness.py

```diff
@@ -249,6 +249,32 @@ def assertValuesOfType(self, valueList, valueType, allowNone=False):
                 continue
             self.assertIsInstance(value, valueType)
 
+    def requireEntityReferencesFixture(self, fixtureName, skipTestIfMissing=False):
+        """
+        A convenience companion to requireFixture, to construct an
+        entity reference object around each raw string in a list
+        fixture.
+        """
+        return [
+            self._manager.createEntityReference(r)
+            for r in self.requireFixture(
+                fixtureName,
+                skipTestIfMissing,
+            )
+        ]
+
+    def requireEntityReferenceFixture(self, fixtureName, skipTestIfMissing=False):
+        """
+        A convenience companion to requireFixture, to construct an
+        entity reference object around a raw string fixture.
+        """
+        return self._manager.createEntityReference(
+            self.requireFixture(
+                fixtureName,
+                skipTestIfMissing,
+            )
+        )
+
```

Comment:
[fix] Good thinking, had this on the list for a while, but can we split this out into its own commit, add tests and release notes (as this is public API and people use these in their own suites).

Reply:
I took the liberty of squashing and rebasing, so inserted a commit for this as 529df33c